genstudio-external-template-app/src/genstudiopem/web-src/src/Constants.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export const extensionId: string = "genstudio-external-template-app";
+export const extensionLabel: string = "External Template App";
+export const LOAD_TEMPLATE_ACTION = "genstudio-external-template-app/get-templates";
+//TODO: Is this required?
+export const ICON_DATA_URI: string =
```

Comment:
Nitpick. The Icon should be different from DAM extension for demo purpose, we can have something simple :D